node/src/main/scala/org/bitcoins/node/models/PeerDAO.scala

```diff
@@ -1,12 +0,0 @@
-package org.bitcoins.node.models
-
-import org.bitcoins.db.{CRUDAutoInc}
-import slick.jdbc.SQLiteProfile.api._
-
-import scala.concurrent.ExecutionContext
-import org.bitcoins.node.config.NodeAppConfig
-
-case class PeerDAO()(implicit ec: ExecutionContext, appConfig: NodeAppConfig)
```

Comment:
Are we removing this table altogether?

Reply:
Yeah, it wasn't implemented. It was essentially a place holder for something that never got implemented.